README.md

```diff
@@ -928,6 +928,23 @@ Other Style Guides
     );
     ```
 
+  <a name="functions--rest-operator-last"></a><a name="7.16"></a>
```

Comment:
we don't add new sections with numbered links; those are legacy.

Reply:
I tried to be consistent with the above points as they had been written. If still their is something wrong then can you help me how can I fixed that.

Reply:
done